src/main/app/claims/models/claim.ts

```diff
@@ -165,6 +166,10 @@ export class Claim {
       return ClaimStatus.MORE_TIME_REQUESTED
     } else if (!this.response) {
       return ClaimStatus.NO_RESPONSE
+    } else if (this.isCCJPaidWithinMonth()) {
+      return ClaimStatus.PAID_IN_FULL_CCJ_CANCELLED
+    } else if (!this.isCCJPaidWithinMonth()) {
```

Comment:
Can just be `else` now?

Reply:
Hmm the `else` beneath it gets thrown off if I turn this `else if` into just an `else`. It expects a declaration or statement

Reply:
Yes, I meant that the `else` we currently have should be impossible to reach, since at least one of the previous conditionals _must_ be satisfied